elements/pfe-accordion/docs/index.md

```diff
@@ -20,33 +20,33 @@ Accordions toggle the visibility of sections of content. They feature panels tha
 
 <pfe-accordion>
   <pfe-accordion-header>
-    <h3>Why do wizards need money if they could just create it?</h3>
+    <h3>Laboris sunt qui dolor consectetur excepteur in aliqua ipsum?</h3>
```

Comment:
@kylebuch8 I updated this to lorem ipsum because Harry Potter's author is not the most inclusive person unfortunately.